java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -228,6 +228,21 @@ public BazelCommand bazelCommand(List<String> args) {
       return new BazelCommand(args);
   }
 
+  /**
+   * Returns a builder for invoking bazel.
+   */
+  public build.bazel.tests.integration.BazelCommand.Builder bazel(String arg, String... args) {
+    return bazel(
+        Stream.concat(Stream.of(arg), Stream.of(args)).collect(Collectors.toList()));
+  }
+
+  /**
+   * Returns a builder for invoking bazel.
+   */
+  public build.bazel.tests.integration.BazelCommand.Builder bazel(List<String> args) {
```

Comment:
here and above: there is no need for the full qualified name for BazelCommand, it is in the same package

Reply:
There is a name clash with WorkspaceDriver.BazelCommand

Reply:
I prefer to not have 2 BazelCommand, I prefer to break the API there (which also relate to moving BazelCommand out of WorkspaceDriver)

Reply:
Ok, I'll do that